content/en/networks/mainnet/details/index.md

```diff
@@ -59,10 +59,13 @@ aliases:
 
 ## Resources
 
+<!-- markdown-link-check-disable -->
 - [Latest chain snapshot (pruned)](https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car)
 - [Latest chain snapshot (full)](https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/complete_chain_with_finality_stateroots_latest.car)
+<!-- markdown-link-check-enable -->
```

Comment:
Link checker flagged https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car 
and https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/complete_chain_with_finality_stateroots_latest.car as 403. Manual check confirms that there is some sort of issue, both urls show a page with:

```
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>E7JVR4Z1HG58VX66</RequestId>
<HostId>0LAIoYqV0NRgZDPYVEZIiDkeq3i39nxmz67Ri6yHOEq/mrP0773VJGdYfqVCUqAsJfi2CmV5O5U=</HostId>
</Error>
```

Disabled link check for these lines to pass pre-commit, not sure what needs to happen with the links

@johnnymatthews

Reply:
Disable these lines and create a new issue to find out the correct snapshot addresses.